lr-cli/cortx_setup/api_spec.yaml

```diff
@@ -67,6 +67,8 @@ cluster:
   config:
     component:
       ClusterConfigComponent
+  reset:
+    NodeResetCluster
```

Comment:
Change section name to `ClusterResetNode`

Reply:
Done